depfinder/inspection.py

```diff
@@ -36,7 +36,7 @@
 from collections import defaultdict
 from typing import Union
 
-from stdlib_list import stdlib_list
+from .stdliblist import builtin_modules
```

Comment:
Could just be me but I find `stdliblist` a bit ugly / hard to read, maybe the module could be `stdlib_list_` instead or something

Reply:
Super ugly! I'm terrible with names. I'm OK with `stdlib_list_`. Let's see what @ericdill thinks and we'll rename.

Reply:
Actually, let's hold this one for a bit. I forgot I'm also a maintainer of stdlib-list and these changes should be there instead.

Reply:
> Let's see what @ericdill thinks and we'll rename.

FWIW you don't need my approval for anything in this project. do whatever you think is best. i trust you on this stuff

Reply:
i'd prefix this to be `_stdlib_list` but it's an internal detail so we can fix it in post